src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -54,25 +54,25 @@ export default {
  the selected dataset snapshots.'),
 
     encryption_placeholder: T('Encryption'),
-    encryption_tooltip: T(''),
+    encryption_tooltip: T('Checking this box, enables additional options for encryption replication'),
 
     encryption_key_generate_placeholder: T('Generate Encryption Key'),
-    encryption_key_generate_tooltip: T(''),
+    encryption_key_generate_tooltip: T('If Hex key type is selected, this will autogenerate an encryption key'),
 
     encryption_key_hex_placeholder: T('Encryption Key'),
-    encryption_key_hex_tooltip: T(''),
+    encryption_key_hex_tooltip: T('Uncheck Generate Encryption Key to import your custom Hex key'),
 
     encryption_key_passphrase_placeholder: T('Passphrase'),
-    encryption_key_passphrase_tooltip: T(''),
+    encryption_key_passphrase_tooltip: T('If Passphrase key type is selected, enter an alphanumeric encryption key'),
```

Comment:
Cleanup suggestion:
"Enter an alphanumeric encryption key. Only available when <i>Passphrase</i> is the chosen key format.